test/ServerlessClamscan.test.ts

```diff
@@ -6,7 +6,7 @@ import { EventBus } from '@aws-cdk/aws-events';
 import { SqsDestination, EventBridgeDestination } from '@aws-cdk/aws-lambda-destinations';
 import { Bucket } from '@aws-cdk/aws-s3';
 import { Queue } from '@aws-cdk/aws-sqs';
-import { Stack } from '@aws-cdk/core';
+import { CfnOutput, Fn, Stack } from '@aws-cdk/core';
```

Comment:
```suggestion
import { Stack } from '@aws-cdk/core';
```

These imports won't be required with the below changes

Reply:
Updated